app/src/main/java/com/ballboycorp/com/topik/SonarBugSample.kt

```diff
@@ -0,0 +1,19 @@
+package com.ballboycorp.com.topik
+
+interface Sample {
+    fun method()
+
+    fun anotherMethod()
+}
+
+class SampleImpl : Sample {
+    override fun method() {
```

Comment:
Add a nested comment explaining why this function is empty or complete the implementation. [author:musooff@gmail.com, component:musooff_Topik_AYB-ZFlS8-xhCYxQvxYf:app/src/main/java/com/ballboycorp/com/topik/SonarBugSample.kt, creationDate:2022-05-01T16:56:35+0900, debt:5min, effort:5min, flows:[], hash:1a2f6c65fe7096cb2f63d80515f89c90, key:AYB-q2G40n16_HoDUh6x, line:10, message:Add a nested comment explaining why this function is empty or complete the implementation., project:musooff_Topik_AYB-ZFlS8-xhCYxQvxYf, quickFixAvailable:false, rule:kotlin:S1186, scope:MAIN, severity:CRITICAL, status:OPEN, tags:[suspicious], textRange:[endLine:12, endOffset:5, startLine:10, startOffset:26], type:CODE_SMELL, updateDate:2022-05-02T16:47:05+0900]